node-backend/app/cronJobs/ResetHi5QuotaBalance.js

```diff
@@ -0,0 +1,6 @@
+var cron = require("node-cron");
+
+const db = require("../models/sequelize");
+cron.schedule(process.env.TIME_RESET_HI5_QUOTA, () => {
```

Comment:
Why this property is not added in sample.env? 
@jitendrasbunde @avi4630 if you are adding/using any new ENV property you should add it to sample.env as well